cmd/root.go

```diff
@@ -181,6 +206,8 @@ func startScanner(namespace string, clusterWide bool, scanner *scanner.Scanner)
 
 	runUID := uuid.New().String()
 	ctx := context.Background()
+	scanner.BeforeScan(ctx)
+	defer scanner.AfterScan(ctx)
```

Comment:
In the consistency mode used to send the health checks to SUSE Observabiliry (REPEAT_SNAPSHOT), it necessary to send a payload to start the snapshot and another to close it. All the request between these two request, will be the health checks send by the audit scanner. 

I've add these functions here just to avoid any the work to handle the concurrency nature of the audit scanner and the need to add some control to avoid multiple start/stop payloads. If we send more then one of this payload, the health checks will not work as expected.